contracts/claimsManagers/ClaimsManagerSingleAccount.sol

```diff
@@ -25,6 +25,7 @@ contract ClaimsManagerSingleAccount {
     	currentInvestigationPeriodEnd = 0;
     	activePayoutEvent = false;
     	governance = msg.sender;
+		status = ClaimsStatus.Ready;
```

Comment:
@stephenbyrne99 can you fix the indentation here?

Reply:
This is not idented locally - not sure why it is on the PR

Reply:
Looks like it might be using tabs instead of spaces, causing the weird indention.